rcore/flash.c

```diff
@@ -134,6 +134,11 @@ static void _fs_read_page_ofs(int pg, size_t ofs, void *p, size_t n) {
     flash_read_bytes(REGION_FS_START + pg * REGION_FS_PAGE_SIZE + ofs, (uint8_t *)p, n);
 }
 
+void flash_read_file_bytes(const File* file, size_t ofs, void *p, size_t n) {
```

Comment:
this probably should be a `fs_read` function -- remember that it has to continuously iterate; this will behave incorrectly for n > `PAGE_SIZE`